Bottom: 090011dc27b03f92db8237be096251c4f964c029
Top:    1944e68a8828835d035efc0d3f9915738829dcde
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-28 03:23:52 +0000

Refresh of add-name-tags-to-contours-and

---

diff --git a/src/polygon.c b/src/polygon.c
index 0a74a22..e290907 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -2331,7 +2331,7 @@ pv_outline_callback (const BoxType * b, void *cl)
     }
 
 #ifdef DEBUG_CIRCSEGS
-  if (!(np = CirclePoly (pv->X, pv->Y, pv->Thickness / 2), feature_name))
+  if (!(np = CirclePoly (pv->X, pv->Y, pv->Thickness / 2, feature_name)))
 #else
   if (!(np = CirclePoly (pv->X, pv->Y, pv->DrillingHole / 2, feature_name)))
 #endif
